doc/modules/linear_model.rst

```diff
@@ -1045,6 +1045,8 @@ The following table summarizes the penalties and multinomial multiclass supporte
 
 The "lbfgs" solver is used by default for its robustness. For large datasets
 the "saga" solver is usually faster.
+For `n_samples >> n_features`, "newton-cholesky" is a good choice. In addition, it can
+reach high precision (tiny `tol` values).
```

Comment:
```suggestion
The "lbfgs" solver is used by default for its robustness. For
`n_samples >> n_features`, "newton-cholesky" is a good choice and can reach high
precision (tiny `tol` values). For large datasets
the "saga" solver is usually faster.
```
I think this reads a bit better because the sentence about SGD then comes after saga as it was meant to be originally, I guess.